src/rules/__tests__/normalizeRuleComponentError.test.js

```diff
@@ -33,4 +33,10 @@ describe('normalizeRuleComponentError', function () {
       new Error('some other error')
     );
   });
+
+  it('returns an error when an object that is not an error is received', function () {
```

Comment:
The real purpose behind our change down in `normalizeRuleComponentError` was to serialize down objects or arrays instead of getting the terrible conversion of `[object Object]`. Perhaps notating that serialization is what's expected in this test name wold help clarify what we're testing here.

Reply:
Yeah, I agree. I'll make the change.